sklearn/preprocessing/callable_transformer.py

```diff
@@ -0,0 +1,44 @@
+from ..base import BaseEstimator, TransformerMixin
+from ..utils import check_array
+
+
+class CallableTransformer(BaseEstimator, TransformerMixin):
+    """Allows the construction of a transformer from an arbitrary callable.
```

Comment:
I suggest a more direct and brief "Constructs a transformer from an arbitrary callable."